src/core_plugins/metrics/public/visualizations/lib/circle_gauge.js

```diff
@@ -0,0 +1,155 @@
+import _ from 'lodash';
```

Comment:
i think this whole lib folder could use further separation to make it clearer what each of the files is for and have only top level files (which are imported by other parts of the code) on this level